datastore/team.go

```diff
@@ -181,10 +183,10 @@ type Team struct {
 }
 
 type requestToJoinTeam struct {
```

Comment:
Should this struct type also be exported?

Reply:
Yes! Done!